templates/Mail/base.html.twig

```diff
@@ -0,0 +1,117 @@
+{% apply inline_css(source('@public/' ~ asset('build/mail.css'))) %}
+    <!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
+            "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
+
+    <html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en">
+    <head>
+        <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
+        <meta name="viewport" content="width=device-width">
+        {% block title %}
+            <title>{{ fallbacks.get('site_title') }}</title>
+        {% endblock %}
+    </head>
+    <body>
+
+    <span class="preheader"></span>
```

Comment:
I have given some feedback in https://app.activecollab.com/108877/projects/533?modal=Task-123073-533 on the template.

Reply:
aangepast in 02c5ac9